ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -445,6 +445,33 @@ def test_tensorflow_log_sigmoid(
     )
 
 
+# reciprocal
+@handle_frontend_test(
+    fn_tree="tensorflow.math.reciprocal",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
According to the [tensorflow doc](https://www.tensorflow.org/api_docs/python/tf/math/reciprocal):
> A Tensor. Must be one of the following types: bfloat16, half, float32, float64, int8, int16, int32, int64, complex64, complex128.

So can you test atleast for ints along with floats. Also, will be super helpful to mention the rest of dtypes TODO comment.